### PR TITLE
Fix sizing of line toolbar

### LINKED ISSUE #412: Toolbar overlaps tabs
image

### PR DESCRIPTION
**Pull request by axelboc**
Fix #412 

This had nothing to do with having `min-width` on the scale selectors as I initially thought. Turns out the `ScaleSelector` component was rendering a fragment with two children, and this wasn't playing well with the responsive `Toolbar`.

### CODE REVIEW

File: src/h5web/toolbar/LineToolbar.tsx [hunk 1]
```diff
@@ -47,8 +47,6 @@ function LineToolbar(props: ToolbarProps): ReactElement {
         onScaleChange={setXScaleType}
       />
 
-      <Separator />
-
```
Comment (axelboc) [inline src/h5web/toolbar/LineToolbar.tsx:51]: I've removed the separator between the two selectors. Can't remember if we had talked about it when you first implemented it. I feel like it's more in line with what we've done so far of "grouping" controls of the same type (single toggle buttons, notably)

Comment (loichuder) [reply src/h5web/toolbar/LineToolbar.tsx:51]: Yes we did. I preferred having a clear separation between the two dropdowns. But in the end, it looks all right without it due to a bigger spacing between the two.

**Review by loichuder — APPROVED** [review]
Nice work :slightly_smiling_face:

**Pull request merged**